WBooks/Helpers/LineTextField.swift

```diff
@@ -10,18 +10,24 @@ import UIKit
 
 class LineTextField: UITextField {
     
+    override var placeholder: String? {
+        didSet {
+            configurePlaceholder()
```

Comment:
Just checking: Isn't there maybe a placeholderFont or something to avoid attributed string and resetting it for each placeholder?

Reply:
All references that i've found, use Attirubted string to set placeholder's font